packages/tina-graphql-gateway/src/auth/TinaCloudProvider.tsx

```diff
@@ -105,12 +108,35 @@ export const AuthWallInner = ({
  *
  * Note: this will not restrict access for local filesystem clients
  */
-export const TinaCloudAuthWall = (props: TinaCloudAuthWallProps) => {
+export const TinaCloudAuthWall = (
+  props: TinaCloudAuthWallProps & CreateClientProps
+) => {
   useTinaAuthRedirect()
+  const cms =
+    props.cms ||
+    new TinaCMS({
+      enabled: true,
+      sidebar: true,
+    })
+  if (!cms.api.tina) {
+    cms.api.tina = createClient(props)
+  }
+  switch (props.media) {
```

Comment:
@jamespohalloran, @jeffsee55 and @perkinsjr.

This will make media extremely easy to setup but are am I hiding too much from the user?

Reply:
The reason we need to call 
```ts
cms.media.store = new store.CloudinaryMediaStore()
```
is because when we merge https://github.com/tinacms/tina-graphql-gateway/pull/279. The API will be 
```ts
cms.media.store = new store.CloudinaryMediaStore(tinaCloudClient)
``` 
and in this easier to setup version the user does not have access the the tinaCloudClient.

Reply:
I don't see the issue with this, I don't believe it's hiding too much from a developer. It also makes extendability and setup a breeze.

Reply:
> I don't see the issue with this, I don't believe it's hiding too much from a developer. It also makes extendability and setup a breeze.
👍  
I agree and we still let the user pass in there own CMS so its still open for extension.

Reply:
Yep agreed 👍